javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -31,6 +31,7 @@
 import com.github.javaparser.ast.observer.AstObserver;
 import com.github.javaparser.ast.observer.ObservableProperty;
 import com.github.javaparser.ast.observer.PropagatingAstObserver;
+import com.github.javaparser.ast.stmt.BlockStmt;
```

Comment:
This bit confused me just now as I was reading through -- turns out this is due to the `{@link}` on line 316. 

Reading up on this, the alternative seems to be to state the fully qualified name instead.

Happy to edit if requested as I don't have a strong view either way (perhaps a small leaning towards keeping the import).